core/src/main/scala/kafka/log/ProducerStateManager.scala

```diff
@@ -472,17 +472,24 @@ class ProducerStateManager(val topicPartition: TopicPartition,
     }
   }
 
+  private def inRangeInclusive(snapshotFile: File, startOffset: Long, endOffset: Long): Boolean = {
+    val offset = offsetFromFilename(snapshotFile.getName)
+    offset >= startOffset && offset <= endOffset
+  }
+
+  private def inRangeExclusive(snapshotFile: File, startOffset: Long, endOffset: Long): Boolean = {
```

Comment:
Perhaps inRangeExclusive and inRangeInclusive could just be outOfRange and inRange?